this tests multiple non-utf8 encodings that musl doesn't support.
--
diff --git a/installed-tests/js/meson.build b/installed-tests/js/meson.build
index 6db887d..8f3e44e 100644
--- a/installed-tests/js/meson.build
+++ b/installed-tests/js/meson.build
@@ -242,7 +242,6 @@ modules_tests = [
     'Console',
     'ESModules',
     'AsyncMainloop',
-    'Encoding',
     'GLibLogWriter',
     'Global',
     'Timers',
